web-app/CKS.Web.Test/IntegrationTests/ResponseHeaderTests.cs

```diff
@@ -0,0 +1,82 @@
+using CKS.Web.Test.IntegrationTests.Infrastructure;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace CKS.Web.Test.IntegrationTests
+{
+	public class ResponseHeaderTests : TestBase
+	{
+		public ResponseHeaderTests(ITestOutputHelper output) : base(output) {}
+
+		[Fact]
+		public async void ByDefaultCorrectCacheControlHeadersAreReturned()
```

Comment:
Can this be something like
```suggestion
		public async void HtmlJsonXmlReturnMustRevalidateNoCacheHeader()
```
or something that more describes the behaviour